.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
+**PLEASE READ:** Before posting this report, *please* carefully read our **Troubleshooting Guide** at https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ and search the issues page here for your error message and problem description, as the great majority of bugs are either duplicates, or can be fixed on the user side with a few easy steps.
+
+Also, if you do post an issue here, please describe step by step what you were doing when the error occured, in as much detail as possible. Otherwise, we will have to close your issue after one week (7 days) if we can't reproduce it. Thanks.
```

Comment:
Also break this single line in multiple ones.

Reply:
See above comment.